app/assets/scripts/components/bar-chart-measurement.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Bar } from 'react-chartjs-2';
+
+export default function BarChart({ title, frequency, xAxisLabels }) {
+  const data = {
+    labels: xAxisLabels,
+    datasets: [
+      {
+        label: 'Measurements',
+        data: frequency,
+        backgroundColor: '#198CFF',
+      },
+    ],
+  };
+
+  const options = {
+    defaultFontFamily:
+      "'Roboto', 'Helvetica Neue', Helvetica, Arial, sans-serif",
+    legend: {
+      display: false,
+    },
+    scales: {
+      yAxes: [
+        {
+          ticks: {
+            beginAtZero: true,
```

Comment:
```suggestion
            beginAtZero: true,
            maxTicksLimit: 5
```

I suggest adding this limit, too many horizontal lines looks messy

Reply:
Great! I couldn't figure out how to do it, thanks!